### ISSUE TITLE
Review resource - Veteran-facing legacy benefit pages on TeamSite: Retirement FAQs 

### ISSUE THREAD

**Issue by lunascoop** [department-of-veterans-affairs/va.gov-team#7602]
Please review the [Veteran-facing legacy benefit pages on TeamSite: Retirement FAQs](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/3c243ef4aeb3e68f14993f8f429764f98a5cfddd/Legacy-benefit-pages-retirement-FAQs-061919.pdf) resource.

### Acceptance Criteria 
- [ ] Confidence with material covered in resource(s)
- [ ] Comment (on this ticket) with any questions or concerns

**Comment by bethpottsVADEPO** [comment]
@lunascoop Hello! I haven't had any luck accessing these resources (pdfs) in the onboarding tickets you assigned to me. Which repo do they live in? I may not have access/permissions to that repo.

**Comment by lunascoop** [comment]
Hi @bethpotts they are in the va.gov-team-sensitive repo. If it helps, I've downloaded and attached the PDF to this ticket.
[Legacy-benefit-pages-retirement-FAQs-061919.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/4781343/Legacy-benefit-pages-retirement-FAQs-061919.pdf)
Also, I've updated the URL redirect info that we discussed last week. Here's the link: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/public-websites/content-team-processes/URL-redirect-process
